js/hoverzoom.js

```diff
@@ -1296,10 +1296,7 @@ var hoverZoom = {
             setTimeout(posImg, options.showWhileLoading ? 0 : 10);
 
             if (options.addToHistory && !chrome.extension.inIncognitoContext) {
-                var url = hz.currentLink.attr('href') || imgDetails.url;
-                if (url.startsWith('/') && url.indexOf('http') < 0)
```

Comment:
@GrosPoulet This was done to fix #283 so can you please make sure you didn't revert that fix?

Reply:
@extesy i checked on Reddit, it works fine (because imgDetails.url is now correctly handled).